packages/snap/src/keyring.test.ts

```diff
@@ -192,6 +210,169 @@ describe('Keyring', () => {
         expectedAddressFromSalt,
       );
     });
+
+    it('should not create an account already in use', async () => {
+      const salt = '0x123';
+      const expectedAddressFromSalt =
+        await simpleAccountFactory.getAccountAddress(
+          await aaOwner.getAddress(),
+          salt,
+        );
+      const account = await keyring.createAccount({
+        privateKey: aaOwnerPk,
```

Comment:
I think it is fine to include in this PR because it relates to the testing suite
Commit: [c229f6d](https://github.com/MetaMask/snap-account-abstraction-keyring/pull/51/commits/c229f6d6c1977909f5bf19107eec5faf8244add8)